src/Aspire.Dashboard/Components/Controls/SummaryDetailsView.razor.cs

```diff
@@ -187,31 +185,42 @@ private void SetPanelSizes(float panel1Fraction)
         _panel2Size = string.Create(CultureInfo.InvariantCulture, $"{(1 - panel1Fraction):F3}fr");
     }
 
-    public async Task OnPageKeyDownAsync(KeyboardEventArgs args)
+    public IReadOnlySet<AspireKeyboardShortcut> SubscribedShortcuts { get; } = new HashSet<AspireKeyboardShortcut>
     {
-        if (_splitterRef is null || !args.OnlyShiftPressed())
+        AspireKeyboardShortcut.ToggleOrientation,
+        AspireKeyboardShortcut.ClosePanel,
+        AspireKeyboardShortcut.ResetPanelSize,
+        AspireKeyboardShortcut.IncreasePanelSize,
+        AspireKeyboardShortcut.DecreasePanelSize
+    };
+
+    public async Task OnPageKeyDownAsync(AspireKeyboardShortcut shortcut)
+    {
+        if (_splitterRef is null)
         {
             return;
         }
 
-        var key = args.Key.ToLower();
-
-        if (key is "t")
+        if (shortcut is AspireKeyboardShortcut.ToggleOrientation)
```

Comment:
Use a switch here?

Reply:
The switch would only have two branches, I don't think it significantly improves readability